### PR TITLE
10.) Cleanup manipulator example.

### PR DESCRIPTION
**Pull request by rnixx**
Number 10 of spitting up the housekeeping PR

### CODE REVIEW

File: examples/manipulator/children.py [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
+from kivy3 import Material
+from kivy3 import Mesh
+from kivy3 import PerspectiveCamera
+from kivy3 import Renderer
+from kivy3 import Scene
 from kivy3.extras.geometries import BoxGeometry
+import math
+import os
+
+"""Demonstrate relative transform of children of a Mesh object."""
```
Comment (AndreMiras) [inline examples/manipulator/children.py]: I think module level docstring are usually on the top of the file (before the imports).
https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring

File: examples/manipulator/children.py [hunk 2]
```diff
@@ -42,7 +41,22 @@
 JOINT_OFFSET_G = [0, 0, 0]
 
 
+def create_joint_rectangle(x, y, z):
+    geometry = BoxGeometry(x, y, z)
+    for i in range(len(geometry.vertices)):
+        geometry.vertices[i][0] += x / 2
```
Comment (AndreMiras) [inline examples/manipulator/children.py]: Untested suggestion:
```suggestion
    for vertice in geometry.vertices:
        vertice[0] += x / 2
```

File: examples/manipulator/children.py [hunk 3]
```diff
@@ -169,14 +183,21 @@ def _rotate_rect(self, dt):
...
+        axis_a_dimensions = (
+            JOINT_SIZE_A[0],
+            JOINT_SIZE_A[2],
+            JOINT_SIZE_A[1]
+        )
         axis_a_geometry = create_joint_rectangle(
-            axis_a_dimensions[0], axis_a_dimensions[1], axis_a_dimensions[2]
+            axis_a_dimensions[0],
+            axis_a_dimensions[1],
+            axis_a_dimensions[2]
```
Comment (AndreMiras) [inline examples/manipulator/children.py]: Untested suggestion:
```suggestion
            *axis_a_dimensions
```
Same applies for other similar calls below

**Review by AndreMiras — APPROVED** [review]
LGTM

File: examples/manipulator/children.py (hunk 1 above)
Comment (rnixx) [reply examples/manipulator/children.py]: merged docsstring with example app docstring

File: examples/manipulator/children.py (hunk 2 above)
Comment (rnixx) [reply examples/manipulator/children.py]: done

File: examples/manipulator/children.py (hunk 3 above)
Comment (rnixx) [reply examples/manipulator/children.py]: done

**Pull request merged**